src/System.ValueTuple/tests/ExtensionsTests.cs

```diff
@@ -5,13 +5,12 @@
 using System;
 using System.Text;
 using Xunit;
-using static ValueTupleTests;
 
 public class ExtensionsTests
```

Comment:
Yeah, basically having the tests in the same namespace + ".Tests" as the type being tested.